### DISCUSSION TITLE
pagy_materialize_nav renders disabled prev/next as empty hyperlinks

### DISCUSSION THREAD

**Discussion by icabodika** [ddnexus/pagy#529] [Q&A] [answered]
Hey there, when I use the pagy_nav helper in my view it renders disabled prev as 
`<span class="page prev disabled">‹&nbsp;Prev</span>`
which works.

When I use pay_materialize_nav helper I get
`<li class="prev disabled"><a href="#"><i class="material-icons">chevron_left</i></a></li>`

The hyperlink calls a root request. Shouldn't it be just?
`<li class="prev disabled"><i class="material-icons">chevron_left</i></li>`

Thanks. Great gem.

**Comment by ddnexus** [comment, marked as answer]
The specific styles (e.g. materialize), generate the markup according to their framework. That should work in a page that uses the specific CSS framework (in that case it's Materialize).

**Reply by icabodika** [reply]
Thanks for amzing fast response. Will use pagy_nav and customise css.
Cheers
